src/istio/chart/templates/tls-validation-job.yaml

```diff
@@ -0,0 +1,95 @@
+# Copyright 2024 Defense Unicorns
```

Comment:
Broad comment on the helm hook approach...I'm wondering what the UX looks like on failure here? Zarf would probably just show a generic helm timeout failure which isn't ideal imo. If we moved this to a zarf action or pepr policy we'd get a more clear error in the zarf/uds output I think.